app/views/tags/edit.html.erb

```diff
@@ -44,7 +44,7 @@
         <% if @tag.can_change_type? %>
           <%= f.select :type, options_for_select(%w(Fandom Character Relationship Freeform UnsortedTag), @tag.type.to_s) %>
         <% else %>
-          <strong><%= t("tag_types.#{@tag.type.tableize.singularize}").capitalize %></strong>
+          <strong><%= ts("%{tag_type}", tag_type: tag_type_label_name(@tag.type.tableize.singularize).singularize) %></strong>
```

Comment:
Is the second `singularlize` intended? Since the `tag_type_label_name` method already calls `singularize`, I think it would be fine to drop it.

Reply:
I can drop both, actually! Good catch.